src/common/components/entry-vote-btn/_index.scss

```diff
@@ -464,3 +465,55 @@ p {
 .reverse-range {
   transform: rotateY(180deg);
 }
+
+datalist {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+
+  @media screen and (max-width: 421px) {
+    display: none;
+  }
+
+}
+.options{
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  justify-content: flex-start;
+}
+
+.options p {
+    font-size: 15px;
+}
+
+.line-slider{
```

Comment:
@MTSaeed As we discussed in our call, there are couple issues still remaining, here in this picture I clicked 75 but got this...
![image](https://user-images.githubusercontent.com/3015990/203338227-fbbd5291-535b-4a3f-8513-48907be1a7ce.png)